docs/policy.json.md

```diff
@@ -0,0 +1,248 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
+
+# Signature verification policy file format
+
+Signature verification policy files are used to specify policy, e.g. trusted keys,
+applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
```

Comment:
I like the idea of users being able to define policy in their home dirs.  We are working on atomic pull --user as we speak for running containers fully without privileges.  But I worry about system policy being overridden by user policy.